mass_mailing_partner/wizard/partner_merge.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2020 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import models
+
+
+class BasePartnerMergeAutomaticWizard(models.TransientModel):
+    _inherit = "base.partner.merge.automatic.wizard"
+
+    def _merge(self, partner_ids, dst_partner=None, extra_checks=True):
+        if dst_partner:
+            contact_ids = self.env["mailing.contact"].search(
+                [("partner_id", "in", partner_ids)]
+            )
+            if contact_ids:
+                contact_ids.sorted(lambda x: 1 if x.partner_id == dst_partner else 0)
```

Comment:
I'm thinking... isn't this returning the sorted result as returned instead of sorting inline? If so, you need to:

```suggestion
                contacts = contact_ids.sorted(lambda x: 1 if x.partner_id == dst_partner else 0)
```

And please, always name properly the variables. It's `contacts`, not `contact_ids`.